tensorflow/compiler/mlir/tools/kernel_gen/tests/tf_64bit_indexer.mlir

```diff
@@ -0,0 +1,43 @@
+// RUN: kernel-gen-opt %s --split-input-file \
```

Comment:
Can we also update the test file name? E.g., jit_i64_indexed_for_large_tensors.mlir

Reply:
name updated.